cmd/get/task_util.go

```diff
@@ -42,18 +43,7 @@ func FetchTaskForName(ctx context.Context, name string, project string, domain s
 }
 
 func FetchAllVerOfTask(ctx context.Context, name string, project string, domain string, cmdCtx cmdCore.CommandContext) ([]*admin.Task, error) {
```

Comment:
Can you please also add examples for new config changes for all the commands touched with filter and sortBy .
Also some documentation on how to reference which fields are available for filtering and sorting should be helpful as not all keys are available to sort or filter.


Synatx on how to filter and what variations of it are available can be added as a common link for all commands and specific examples using the object can be added for each.